warn/scrapers/co.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+from .. import utils
+from ..cache import Cache
+from bs4 import BeautifulSoup
+import re
+from typing import List
+
+__authors__ = ["anikasikka"]
+__tags__ = ["html"]
+__source__ = {
+    "name": "Colorado Department of Labor and Employment",
+    "url": "https://cdle.colorado.gov/employers/layoff-separations/layoff-warn-list",
+}
+
+
+def scrape(
+    data_dir: Path = utils.WARN_DATA_DIR,
+    cache_dir: Path = utils.WARN_CACHE_DIR,
+) -> Path:
+    """
+    Scrape data from Colorado.
+
+    Keyword arguments:
+    data_dir -- the Path were the result will be saved (default WARN_DATA_DIR)
+    cache_dir -- the Path where results can be cached (default WARN_CACHE_DIR)
+
+    Returns: the Path where the file is written
+    """
+    # Grab the page
+    page = utils.get_url("https://cdle.colorado.gov/employers/layoff-separations/layoff-warn-list")
+    html = page.text
+
+    # Write the raw file to the cache
+    cache = Cache(cache_dir)
+    cache.write("co/main/source.html", html)
+
+    # Setting the headers to centralize among archived and new data
+    cleaned_data = []
+
+    # Parses the current year's data into a CSV file
+    soup = BeautifulSoup(html, "html5lib")
+    current_link = str(soup.find("a", class_="btn btn-primary")).split("\"")[3]
+
+    current_page = utils.get_url(current_link)
+    current_html = current_page.text
+
+    soup_current = BeautifulSoup(current_html, "html5lib")
+
+    for rows in soup_current.find(class_="waffle").find_all("tr"):
+        vals: List[str] = []
+        scrape_spreadsheet(rows, vals)
+        if (len(vals) == 0):
+            continue
+        cleaned_data.append(vals)
+
+    # Goes through the accordion links to get past data
+    main = soup.find_all('dl')
+    for item in main:
+        my_list = item.find('ul')
+        for li in my_list.find_all('li'):
+            link = (str)(li).split("\"")[1]
```

Comment:
What is this line doing?

Reply:
I'm not sure why `(str)(li)` is necessary. I think the more traditional code would `str(li)`.